### ISSUE TITLE
playback speed for Voice Messages

### ISSUE THREAD

**Issue by ArchiveChan** [Vencord/plugin-requests#662]
### Plugin Description

ability to change playback speed for Voice Messages by clicking new button.
And maybe add plugin setting for default playback speed (eg i need all messages to always be at x1.25 speed)

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Feathers8** [comment]
It seems like this PR https://github.com/Vendicated/Vencord/pull/2530 would fulfill this request, if it were to get merged.

**Comment by D3SOX** [comment]
Yup @Feathers8 thanks for bringing this up, I didn't know about this issue before. The ability to set a default playback speed for different types of embeds is a great idea!

**Comment by D3SOX** [comment]
@ArchiveChan Added options to set the default speeds for all 3 types
